DESCRIPTION

```diff
@@ -1,9 +1,11 @@
 Package: tempConverter
 Title: Temperature Converter
-Suggests: testthat
+Suggests: testthat,
+          covr
 Version: 0.0.0.9000
-Maintainer: Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk> 
-Author: Alexander Konovalov <alexander.konovalov@st-andrews.ac.uk>, Laura Marshall <lhm@st-and.ac.uk>
+Authors@R: c(
```

Comment:
We can leave this as is, but these are unrelated changes in one commit - ideally should be two different commits.

Reply:
good to know... you'll get me doing tidy github repo's yet! :)

Reply:
yes, and even furthermore, changing authors is unrelated to the topic of the PR (code coverage) so I'd be reluctant to do "squash and merge" as well...